data/init.sql

```diff
@@ -9,8 +9,6 @@ CREATE TABLE IF NOT EXISTS ciraconfigs(
       mps_port integer,
       user_name varchar(40),
       password varchar(63),
```

Comment:
these changes also need to be made over in open-amt-cloud-toolkit

Reply:
Changes made just forgot to make a pr